<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Direct_Multi_Screen</class>
 <widget class="QWidget" name="HNS_Direct_Multi_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="PTE_Input">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Cancel">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>710</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="PTE_Errors">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="HNS_Signboard_Preview" name="GV_Preview" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_OK">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>710</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Signboard_Preview</class>
   <extends>QWidget</extends>
   <header>hns_signboard_preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
